commons/feature_toggles/feature_toggle_manager.py

```diff
@@ -0,0 +1,69 @@
+import os
+import json
+from time_tracker_api.security import current_user_email
+from azure.appconfiguration import AzureAppConfigurationClient
+
+
+class MSConfig:
+    def check_variables_are_defined():
+        auth_variables = ['AZURE_APP_CONFIGURATION_CONNECTION_STRING']
+        for var in auth_variables:
+            if var not in os.environ:
+                raise EnvironmentError(
+                    "{} is not defined in the environment".format(var)
+                )
+
+    check_variables_are_defined()
+    AZURE_APP_CONFIGURATION_CONNECTION_STRING = os.environ.get(
```

Comment:
I know this is not in your ticket scope. But if you find some way to improve this class, let us know.

Reply:
We have changed our class that function, Roberto is going to help us to refactor this specific class, hopefully, it will be ready today if not we will create a ticket.